### DISCUSSION TITLE
How to skip logout in serenity Java

### DISCUSSION THREAD

**Discussion by sapnakalokar** [serenity-bdd/serenity-core#3210] [General]
I want to complete test suite in one go where I will login only once and my my all test cases will execute and after finishing all test cases browser will quit.

**Comment by globalworming** [comment]
could you elaborate? you'd like to keep the browser open so you stay logged in?

you can see if https://serenity-bdd.github.io/docs/reference/serenity-properties#serenityuseuniquebrowser works for you, or you tell us more about how you actually authenticate, e.g. if it's cookie based,  then a way to skip login is to start your browser and set the authentication cookie.

**Comment by sapnakalokar** [comment]
In my project there is one time password requires to login application. So, I am planning to run one feature file (which contains 15-20 test cases) in one single login. Once all test cases are executed from one feature file then browser should close. and start a new browser for another feature file.
